core/src/lib.rs

```diff
@@ -291,12 +339,29 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    /// Returns an symbolic representation of the schema suitable for applying across Mentat stores.
+    pub fn to_edn_value(&self) -> edn::Value {
+        edn::Value::Vector((&self.schema_map).into_iter()
```

Comment:
I'm surprised that this works — you're moving out of a borrowed thing. `into_iter` consumes it`self`.